lib/wsdl.js

```diff
@@ -899,6 +900,16 @@ ElementElement.prototype.description = function(definitions, xmlns) {
         if (this.$ref) {
           element = definitions.descriptions.types[typeName];
         }
+        else if(type.namespace
+            && type.namespace in xmlns
+            && definitions
+            && definitions.schemas
+            && definitions.schemas[xmlns[type.namespace]]
+            && definitions.schemas[xmlns[type.namespace]].complexTypes
+            && definitions.schemas[xmlns[type.namespace]].complexTypes[typeName]) {
```

Comment:
The version of lodash used in this project is 2.x, and there is no `_.get` in this version, and `_.has` doesn't accept paths as parameters.